src/libraries/System.Net.Sockets/src/System/Net/Sockets/MulticastOption.cs

```diff
@@ -25,7 +25,7 @@ public MulticastOption(IPAddress group, IPAddress mcint)
                 throw new ArgumentNullException(nameof(mcint));
             }
 
-            Group = group;
+            _group = group;
```

Comment:
Why this change?

Reply:
Because compiler complains that the field `_group ` not initialized